### DISCUSSION TITLE
Question about the gravity field model for the Variational equations

### DISCUSSION THREAD

**Discussion by RoyanLiSu** [groops-devs/groops#164] [Q&A] [answered]
Hi Developers,
 the Variational equations model uses the GOCO06s as a static gravity field model. the input models are:
GOCO06s_aodReduced_static_2010-01-01.gfc
GOCO06s_aodReduced_trend_2010-01-01.gfc
GOCO06s_aodReduced_annualCos_2010-01-01.gfc
GOCO06s_aodReduced_annualSin_2010-01-01.gfc
 if I want to use another gravity field model, for example, ITSG do I have to reduce the effect of AOD from the static, the trend component, and annual oscillation?

**Comment by tmayerguerr** [comment]
Dear  RoyanLiSu,

For the orbit calculation you usually want the full gravity field signal. The model GOCO06s_aodReduced contains all signals except AOD. Therefore the AOD product is added in PreprocessingVariational. In all GRACE monthly solutions (like ITSG-Grace) AOD is also not included and must be added extra. For other gravity field models you have to read the corresponding documentation.

regards
Torsten

**Comment by MyYouth11** [comment]
Hello, how can I get this data, or how can I process data to get it?
GOCO06s_aodReduced_static_2010-01-01.gfc
GOCO06s_aodReduced_trend_2010-01-01.gfc
GOCO06s_aodReduced_annualCos_2010-01-01.gfc
GOCO06s_aodReduced_annualSin_2010-01-01.gfc

**Comment by tmayerguerr** [comment]
Hi,

You can estimate the static, trend, and annual components from the AOD1B timeSplines [gravityfield->timeSplines](https://groops-devs.github.io/groops/html/gravityfieldType.html#timeSplines) using [Gravityfield2TrendPotentialCoefficients](https://groops-devs.github.io/groops/html/Gravityfield2TrendPotentialCoefficients.html) and subtract the the coefficients with [Gravityfield2PotentialCoefficients](https://groops-devs.github.io/groops/html/Gravityfield2PotentialCoefficients.html).

We will publish the new GOCO2025s in the next weeks. For the new model we will provide AOD1B_RL06_included_in_GOCO2025s.gfc and AOD1B_RL07_included_in_GOCO2025s.gfc.

regards
Torsten

**Comment by MyYouth11** [comment]
Thank you so much for your patient reply and I am looking forward to it, I will follow this.



---- Replied Message ----
| From | Torsten ***@***.***> |
| Date | 06/06/2025 17:25 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [groops-devs/groops] Question about the gravity field model for the Variational equations (Discussion #164) |

Hi,

You can estimate the static, trend, and annual components from the AOD1B timeSplines gravityfield->timeSplines using Gravityfield2TrendPotentialCoefficients and subtract the the coefficients with Gravityfield2PotentialCoefficients.

We will publish the new GOCO2025s in the next weeks. For the new model we will provide AOD1B_RL06_included_in_GOCO2025s.gfc and AOD1B_RL07_included_in_GOCO2025s.gfc.

regards
Torsten

**Comment by tmayerguerr** [comment]
GOCO2025s is now available at https://doi.org/10.3217/f48p8-8h651

**Reply by ReubenLi1999** [reply]
[scenario1.zip](https://github.com/user-attachments/files/21328559/scenario1.zip)
Hello developer,

Based on your description, I make such GROOPS script to subtract the static part of AOD1B RL07 from GOCO2025s, and get the static part of GOCO2025s_aodReduced. May you check if it is correct or not? 
Thanks very much!

Best regards,
Haosi

**Comment by tmayerguerr** [comment, marked as answer]
Hi Haosi,

As I see your script is correct.

To extract also the temporal parts, use [Icgem2PotentialCoefficients](https://groops-devs.github.io/groops/html/Icgem2PotentialCoefficients.html) with the attached [groopsConfigGoco.zip](https://github.com/user-attachments/files/21564541/groopsConfigGoco.zip) and apply your script to all parts.

Best regards
Torsten

**Reply by ReubenLi1999** [reply]
Thank you very much!
